examples/scenes/src/download.rs

```diff
@@ -68,12 +69,13 @@ impl Download {
                             builtin.license,
                             builtin.info
                         );
+                        total_bytes += builtin.expected_size;
                     }
 
                     // For rustfmt, split prompt into its own line
-                    const PROMPT: &str =
-                "Would you like to download a set of default lottie files, as explained above?";
-                    accepted = Confirm::new(PROMPT).with_default(false).prompt()?;
+                    let prompt =
+                        format!("Would you like to download a set of default lottie files, as explained above? ({} bytes)", total_bytes);
```

Comment:
```suggestion
                        format!("Would you like to download a set of default lottie files, as explained above? ({})", byte_unit::Byte::from_bytes(total_bytes.into()).get_appropriate_unit(false));
```
(but formatted properly)

Reply:
Nice!! 
`? Would you like to download a set of default lottie files, as explained above? (24.71 MB) (y/N)`